horovod/runner/launch.py

```diff
@@ -273,10 +316,18 @@ def parse_args():
                              'HOROVOD_START_TIMEOUT can also be used to '
                              'specify the initialization timeout.')
 
-    parser.add_argument('--network-interface', action='store', dest='nics',
+    parser.add_argument('--network-interfaces', action=make_nic_action(deprecated=False), dest='nics',
                         help='Network interfaces that can be used for communication separated by '
                              'comma. If not specified, Horovod will find the common NICs among all '
```

Comment:
Should it be singular "common NIC" here or should it be "use them" a bit later in the sentence?

Reply:
good point, I have simplified that bit

Reply:
great!